src/modules/Header/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./styles/header.scss";
+import { NavLink, useLocation } from "react-router-dom";
+import sprite from "./assets/svg/sprite.svg";
+import Weather from "./components/Weather";
+import NavBar from "./components/NavBar";
+import Hamburger from "./components/Hamburger";
+import UserBar from "./components/UserBar";
+import RoutesList from "../../routes";
+
+function Header() {
+  const location = useLocation();
+  const headerClassList = ["header"];
+  const headerContainerClassList = ["header__container"];
+  switch (location.pathname) {
```

Comment:
it would be better to use an object outside this function, something like this:
```
const classes: SHOULD_USE_INTERFACE = {
  [RoutesList.default]: {
     headerClass: "header--default";
     headerContainerClass: "header__container--default";
  }
}

function Header() {
  ...
  headerClassList.push(classes[location.pathname].headerClass)
  headerContainerClassList.push(classes[location.pathname].headerContainerClass)
  ...
}
```

Reply:
done